src/App.tsx

```diff
@@ -150,7 +150,7 @@ export default function App() {
       }
     } else if (outputType === 'dataRequirement') {
       if (measureFile.content) {
-        const { results } = Calculator.calculateDataRequirements(measureFile.content);
+        const { results } = await Calculator.calculateDataRequirements(measureFile.content);
```

Comment:
```suggestion
        const { results } = await Calculator.calculateDataRequirements(measureFile.content, options);
```

With recent changes, we allow the calculation options to be a second arg to the data requirements call now (to allow for measurement period), so let's add the second argument here